test/datasource/opam.spec.ts

```diff
@@ -0,0 +1,225 @@
+import * as fs from 'fs';
+
+import _got from '../../lib/util/got';
+import { getPkgReleases } from '../../lib/datasource/opam';
+
+const got: any = _got;
```

Comment:
```suggestion
const got = mocked(_got);
```

import `mocked` from `../util`